docs/user-guide/get-started.md

```diff
@@ -55,6 +63,18 @@ npx stylelint "**/*.scss"
 
 This config includes the [postcss-scss syntax](https://github.com/postcss/postcss-scss), configures the [built-in rules](../user-guide/rules/list.md) for SCSS, and includes the [stylelint-scss plugin](https://www.npmjs.com/package/stylelint-scss) (a collection of rules specific to SCSS).
 
+If you use Prettier alongside Stylelint, you should also their [SCSS shared config](https://github.com/prettier/stylelint-config-prettier-scss):
+
+```json
+{
+  "extends": [
+    "stylelint-config-standard-scss",
+    "stylelint-config-prettier",
+    "stylelint-config-prettier-scss"
```

Comment:
[suggest] `stylelint-config-prettier` may be unnecessary, since `stylelint-config-prettier-scss` internally extends it.

```suggestion
    "stylelint-config-prettier-scss"
```

See https://github.com/prettier/stylelint-config-prettier-scss/blob/v0.0.1/src/index.js#L4

(BTW, I first learned about `stylelint-config-prettier-scss` here. 😄 )